_data/events.yml

```diff
@@ -99,3 +112,59 @@
     have been involved in it for decades. The conference has a culture of
     critical thought, deep technical insights, continuous improvement, and
     innovation
+- title: 'Training: Foreman (German)'
+  type: 'Training'
+  tags:
+  - training
+  location: Netways, Deutschherrenstr. 15-19, Nuremberg, Germany
+  speaker: 'Dirk G&ouml;tz'
+  start: 2016-07-26 09:00 CEST
+  end: 2016-07-27 17:00 CEST
+  url: 'https://www.netways.de/events_schulungen/foreman/'
+  description: |
+    In this training you will get to know the main architecture behind Foreman.
```

Comment:
This wording reads a little weird, but I'll change it on merge. Thanks for this!